x-pack/plugins/security_solution/cypress/tasks/create_new_rule.ts

```diff
@@ -191,6 +208,33 @@ export const expectAboutFormToRepopulateAndContinue = (rule: CustomRule) => {
   cy.get(ABOUT_CONTINUE_BTN).should('not.exist');
 };
 
+export const expectDefineStepForm = (rule: CustomRule) => {
```

Comment:
@MadameSheema what do you think of defining these (hopefully) reusable expectation functions?

1. Is there utility in naming/exporting these for reuse as opposed to inlining them in tests?
2. Should they live in their own folder (e.g. `/cypress/assertions/`) ?

Reply:
Not using functions as expectations was a team agreement. When I refactored the tests for the first time I used your approach but the team prefers to don't have the expectations wrapped because if there is a failure with the assertions they don't have to dig into the code to understand what is failing. 

Anyway it is a topic we can revisit again :)

Reply:
Inlined expectations here: https://github.com/elastic/kibana/pull/77090/commits/b174f442867774ebe0e41cd59dd9baa6c4014e45